### DISCUSSION TITLE
Multiple Repositories in one component

### DISCUSSION THREAD

**Discussion by JonasDeichelmann** [WeblateOrg/weblate#6318] [Q&A]
We have a WebApp, an iOS App and an Android App which all need the same localisation strings. They are all in separate Repositories on GitHub.
We want to link multiple repositories to one translation component, when we make a localisation change on one point, it should be pushed to all three repositories. 

What is the recommended way of doing that?
Is there a way to connect all three repos to one component?

**Comment by JonasDeichelmann** [comment]
@nijel Do you have any idea?

**Comment by nijel** [comment]
The translation propagation works across components, check out our documentation

**Reply by JonasDeichelmann** [reply]
Thank you for your response. But we want to link one component to multiple GitHub Repositories (iOS, Android, Web).
What is the recommended way of doing that?

**Reply by nijel** [reply]
There is no way to do that - one component = one translation in multiple languages. You need a component for iOS, Android and Web. The translation propagation will help you in keeping these in sync, see https://docs.weblate.org/en/latest/admin/continuous.html#translation-propagation
